core/include/detray/intersection/detail/trajectories.hpp

```diff
@@ -347,6 +354,9 @@ class helix {
     /// B field
     vector3 const *_mag_field;
 
+    /// B field strength
+    scalar _B;
```

Comment:
How does saving this in a a data member work in inhomogeneous fields? Is B not dependent on s?

Reply:
Nvm, I typed before thinking :D

Reply:
Technically we are not supposed to use helix in the inhomogeneous fields. 

In case we have to, it might be enough to take the b field strength at the initial point of trajectories.